src/server/scripts/Northrend/FrozenHalls/ForgeOfSouls/boss_bronjahm.cpp

```diff
@@ -19,15 +19,19 @@
 #include "ScriptedCreature.h"
 #include "SpellScript.h"
 #include "SpellAuraEffects.h"
+#include "CreatureTextMgr.h"
 #include "forge_of_souls.h"
 
-enum Yells
+#define SOUND_DEATH        16598
```

Comment:
Just a note: this line won't be needed once you use creature_text for his death sound.